src/Mount.cpp

```diff
@@ -1281,13 +950,19 @@ void Mount::guidePulse(byte direction, int duration) {
 // The first time with EAST, second with WEST and then with 0.
 /////////////////////////////////
 void Mount::runDriftAlignmentPhase(int direction, int durationSecs) {
-  // Calculate the speed at which it takes the given duration to cover 400 steps.
-  float speed = 400.0 / durationSecs;
+
+  float const numArcMinutes(5.3);
```

Comment:
what is this?

Reply:
400 steps was hardcoded but didn't take into account the current microstepping mode. The 5.3 arc-minutes is from the function documentation which states 400 steps is 5.3 arc-minutes. So now we get 5.3 arc-minutes of motion regardless of microstepping mode. I don't know how 5.3 arc-minutes was chosen :-(.

Reply:
weird ... don't know either what we should do about this one at the moment (if at all)

Reply:
It's just an arbitrary distance that can be used with a "normal" lens (like 100-200mm) that causes star trails about 1/5 of the camera view.